src/client/views/settings-view/cluster-edit/cluster-edit.tsx

```diff
@@ -105,12 +96,14 @@ export class ClusterEdit extends React.Component<ClusterEditProps, ClusterEditSt
     var makeTextInput = ImmutableInput.simpleGenerator(newInstance, this.delegate.onChange);
     var makeDropDownInput = ImmutableDropdown.simpleGenerator(newInstance, this.delegate.onChange);
 
-    var isDruid = newInstance.type === 'druid';
     var needsAuth = ['mysql', 'postgres'].indexOf(newInstance.type) > -1;
 
     return <form className="general vertical">
+      {makeLabel('title')}
+      {makeTextInput('title', /.*/, true)}
+
       {makeLabel('host')}
-      {makeTextInput('host', IP_REGEX, true)}
+      {makeTextInput('host', IP_REGEX)}
```

Comment:
host does not have to be an ip address. it could be `loclahost` or `localhost:8082` or `this-is-where-i.keep-my-druid.com:12345`